alembic/versions/2023_09_07_1927-b33a5b72da8b_image_fourcorners.py

```diff
@@ -0,0 +1,58 @@
+"""image_fourcorners
+
+Revision ID: b33a5b72da8b
+Revises: e78c1e8bec33
+Create Date: 2023-09-07 19:27:39.774329
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'b33a5b72da8b'
+down_revision = 'e78c1e8bec33'
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('images', sa.Column('ra00', sa.REAL(), nullable=False))
```

Comment:
REAL is 32-bit

For the corners of the image, that should be good enough, and since there are eight of these fields, it's worth using the smaller size.

(32-bit floats have 23 bits in the mantissa.  At an angle of 360 degrees, 360/2^23 = about 0.15 arcseconds.  That's not good enough for positions of objects (it is for LS4, but not for everything), so we want to use doubles in SpatiallyIndexed, but it's plenty good enough to decide if an image overlaps a spot on the sky.)